datalad_metalad/indexers/jsonld.py

```diff
@@ -0,0 +1,120 @@
+import os
+from typing import Any, Dict, List, Union
+
+from datalad.metadata.indexers.base import MetadataIndexer
+
+
+class JsonLdTags:
+    ID = '@id'
+    TYPE = '@type'
+    LIST = '@list'
+    GRAPH = '@graph'
+    CONTEXT = '@context'
+
+
+class JsonLdProperties:
```

Comment:
I'm a bit confused by the class name. I might well be wrong, but aren't those terms like `accountablePerson` referring to a particular vocab? Probably schema.org? Think that should be reflected in the class name or at least a docstring.

Reply:
Now, that I'm reviewing PR #62, I realize those classes are meant to represent/enumerate properties and types that are actually used for a particular purpose (here: indexer, there: particular extractor).
However, naming is still a bit unfortunate I think. Having classes of the same name throughout metalad seems bound to cause trouble. Maybe prefix with context: `IndexerJsonLdTags`/`StudyminimetaJsonLdTags` (and -`Types`)?

Reply:
> I'm a bit confused by the class name. I might well be wrong, but aren't those terms like `accountablePerson` referring to a particular vocab? Probably schema.org? Think that should be reflected in the class name or at least a docstring.

You are right. Those should be SchemaOrgProperties, or something similar.



> Now, that I'm reviewing PR #62, I realize those classes are meant to represent/enumerate properties and types that are actually used for a particular purpose (here: indexer, there: particular extractor).
> However, naming is still a bit unfortunate I think. Having classes of the same name throughout metalad seems bound to cause trouble. Maybe prefix with context: `IndexerJsonLdTags`/`StudyminimetaJsonLdTags` (and -`Types`, - `Properties`)?

The idea was to migrate from literal strings to python "constants" (there are a few reasons for that, top-most is to prevent bugs due to misspelling), So the end-goal would be to actually have only one JsonLdTags-class, one SchemOrgTypes-class, and one SchemaOrgProperties-class etc. that are used throughout datalad and its extensions. Not sure what is the best path towards this goal. In order to keep this PR "small", I opted for keeping the string constant classes close to the code that uses them.

Reply:
Done